public_suffix_list.dat

```diff
@@ -10150,6 +10150,10 @@ graphox.us
 // Submitted by Ofer Kalaora <postmaster@activetrail.com>
 activetrail.biz
 
+// Adaptable.io : https://adaptable.io
+// Submitted by Mark Terrel <mark@adaptable.io>
```

Comment:
Please provide a non-personal e-mail address.

Reply:
Updated to support@adaptable.io